webapp/cypress/integration/createBoard.ts

```diff
@@ -112,4 +112,45 @@ describe('Create and delete board / card', () => {
         cy.get('.DeleteBoardDialog button.danger').click({force: true})
         cy.contains(boardTitle).should('not.exist')
     })
+
+    it('scrolls the kanban board when dragging card to edge', () => {
+        cy.visit('/')
+
+        // Create empty boards
+        cy.log('**Create new empty board**')
+        cy.contains('+ Add board').click({force: true})
+        cy.contains('Empty board').click({force: true})
+
+        // Create 10 empty groups
+        cy.log('**Create new empty groups**')
+        for (let i = 0; i < 10; i++) {
+            cy.contains('+ Add a group').click({force: true})
+            cy.get('.KanbanColumnHeader .Editable[value=\'New group\']').should('have.length', i + 1)
+        }
+
+        // Create empty card in first group
+        cy.log('**Create new empty card in first group**')
+        cy.contains('+ New').click({force: true})
+        cy.get('.Dialog').should('exist')
+        cy.get('.Dialog .wrapper').click({force: true})
+        cy.get('.KanbanCard').should('exist')
+
+        // Drag card to right corner and expect scroll to occur
+        cy.get('.Kanban').invoke('scrollLeft').should('equal', 0)
```

Comment:
Looks like something is re-rendering in the background that's causing the first instance of the card to be "detached". Could you add wait time and see if it will fix the failed test?
```suggestion
        cy.get('.Kanban').invoke('scrollLeft').should('equal', 0).wait(1000)
```